test/e2e/deployment.go

```diff
@@ -654,11 +645,16 @@ func testRollbackDeployment(f *Framework) {
 	err = c.Extensions().Deployments(ns).Rollback(rollback)
 	Expect(err).NotTo(HaveOccurred())
 
+	// Wait for the deployment to start rolling back
```

Comment:
We probably should retry the rollback request, in case it fails for some reason.